tfhe/src/shortint/server_key/mod.rs

```diff
@@ -1164,11 +1171,9 @@ impl ServerKey {
     pub fn create_trivial_assign(&self, ct: &mut Ciphertext, value: u64) {
         let modular_value = value % self.message_modulus.0;
 
-        let delta = (1_u64 << 63) / (self.message_modulus.0 * self.carry_modulus.0);
-
-        let shifted_value = modular_value * delta;
-
-        let encoded = Plaintext(shifted_value);
+        let encoded = self
+            .encoding(PaddingBit::Yes)
+            .encode(Cleartext(modular_value));
```

Comment:
Style suggestion: move `Cleartext` wrapping in modular_value definition
Applies to many places (some being with `Plaintext` wrapping before decoding instead)

Reply:
It does apply for some cases but for others not that much